Gemfile

```diff
@@ -20,6 +20,7 @@ gem 'jquery-turbolinks'
 gem 'newrelic_rpm'
 gem 'omniauth-myusa', git: 'https://github.com/18F/omniauth-myusa.git'
 gem 'paperclip'
+gem 'paper_trail', '~> 4.0.0.rc'
```

Comment:
I added the version per the gem's installation instructions, not for any other reason. I will go ahead and remove this.